base/applications/rapps/include/rosui.h

```diff
@@ -495,7 +495,10 @@ class CUiWindow :
 
     virtual ~CUiWindow()
     {
-        T::DestroyWindow();
+        if (::IsWindow(T::m_hWnd))
```

Comment:
```suggestion
        if (T::IsWindow())
```

Reply:
On second thought:
Are you sure this is needed?

Reply:
I think yes. because one can destroy the window directly without calling ~CUiWindow. this will lead to ```m_hWnd``` an invalid hwnd. And if delete is performed after the window is destroyed, trying to delete an invalid hwnd will cause an assert

Reply:
yeah but you probably shouldnt destroy the window without the CUiWindow...

Reply:
sometimes it's just out of your control I think....? 🤣
for example, a parent window will automatically destroy all child window when it's destroyed.
and for top-level window, if you click the close button (unless you handle WM_CLOSE yourself) the system will destroy it too.